src/repository/rest/rickandmortyapi.go

```diff
@@ -0,0 +1,88 @@
+package rest
+
+import (
+	"encoding/json"
+	"github.com/go-resty/resty/v2"
+	"github.com/spf13/viper"
+	"golang-bootcamp-2020/domain/model"
+	_errors "golang-bootcamp-2020/utils/error"
+)
+
+var (
+	apiCharacters = "https://rickandmortyapi.com/api/character/"
+)
+
+type restResponse struct {
+	Info    paginationInfo `json:"info"`
+	Results []interface{}  `json:"results"`
+}
+
+type paginationInfo struct {
+	Next string `json:"next"`
+}
+
+type rickAndMortyApi struct {
+}
+
+type RickAndMortyApiRepository interface {
```

Comment:
Add a description to this exported interface.
Also, change the description in the other functions following the convention